resources/python/ci/ultilint.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+set -eu
+
+USE_DUMMY_DBUS=true PYTHONPATH=:../dbus-interface-lib:../charon:../libSmeagol:../libsmeagol:../marvin-service/src:./ python3 scripts/util/lint_machine_definitions_to_python.py
```

Comment:
These paths don't exist in my development environment. Do they need to be added as requirements in the documentation?